moped-database/README.md

```diff
@@ -51,7 +51,7 @@ https://hasura.io/docs/1.0/graphql/core/hasura-cli/install-hasura-cli.html
 #### 3. The Hasura Cluster Helper
 
 There is a tool we've created that has a few shortcuts available, it's called [hasura-cluster](https://github.com/cityofaustin/atd-moped/blob/main/moped-database/hasura-cluster).
-This just a bash script that runs a few docker-compose commands.
```

Comment:
I don't know why but the old `This just a bash script` typo made me laugh real hard. 😅